pkg/executor/fscache/poolcache.go

```diff
@@ -74,6 +75,7 @@ type (
 	response struct {
 		error
 		allValues    []*FuncSvc
+		FnSvcGroup   map[string]*funcSvcGroup
```

Comment:
Lets not do this. We should add LOG request whereever required.

Channel should receive io.Writer interface type and we can write data via that writer.

That we can even keep same file across executor type.

Reply:
done